src/playground-code-editor.ts

```diff
@@ -677,36 +677,38 @@ export class PlaygroundCodeEditor extends LitElement {
   ): TemplateResult | string {
     const markedObjectName = objectName ?? '';
     const matches = completionData?.matches ?? [];
-    let padding = 0;
     if (matches.length <= 0) {
       // In the situation, that none of the input matches with the
       // completion item suggestion, we exit early, leaving the objectName unmarked.
       return markedObjectName;
     }
 
+    let padding = 0;
     const markedObjectHTML = html`
-      ${matches.map(
-        (match) => html`
-          ${match.indices.map((ind) => {
```

Comment:
Ah I see, so the issue is that multiple indices would match. And fix is to take the first one.